analyze_logs/src/analyzer.py

```diff
@@ -21,8 +22,14 @@
 class Analyzer:
   def __init__(self):
     self.log = Log()
+    self.log.info("Storage account: {0}".format(config.AZURE_STORAGE_ACCOUNT_NAME))
```

Comment:
This should be a debug rather than an info. No biggie, just mentioning it for feedback. It can be changed after merge.